client/src/components/DataFiles/DataFilesBreadcrumbs/DataFilesBreadcrumbs.js

```diff
@@ -113,6 +113,17 @@ const DataFilesBreadcrumbs = ({
 
   return (
     <div className={`breadcrumbs ${className}`}>
+      {scheme === 'projects' && (
+        <>
+          <Link
+            className="breadcrumb-link"
+            to={`/workbench/data/${api}/${scheme}/`}
+          >
+            Shared Workspaces
+          </Link>{' '}
+          /{' '}
+        </>
+      )}
```

Comment:
Maybe stick this under `root` under line [107](https://github.com/TACC/Frontera-Portal/pull/307/files#diff-ade42bc7052b815d608414de9505cf4f4d6919ca0a994873c381f3367a155db5R107)?

Reply:
Not sure I can. Root computes an actual file storage root name for inclusion in a BreadCrumbLink with api, scheme, etc., and the Shared Workspaces link is not really a storage system.

Reply:
Disregard, had mistakenly read `BreadcrumbLink` as `Link`